src/d8/drupal.make.yml

```diff
@@ -0,0 +1,21 @@
+core: 8.x
+api: 2
+
+# Common attributes.
+defaults:
+  projects:
+    subdir: "contrib"
+
+projects:
+  drupal:
+    version: ~
+  features:
+    version: ~
+  field_group:
+    version: ~
+  panels:
+    version: ~
+  pathauto:
+    version: ~
+  token:
+    version: ~
```

Comment:
So, not having the master module is a pretty big deal for Aquifer (see settings.php file) and it's a big part of how Aquifer works. Before we merge this in and call it done, we will probably need to port the Master module to D8.